### ISSUE TITLE
net::ERR_CONNECTION_REFUSED when loading the /telescope page

### ISSUE THREAD

**Issue by jvbalcita** [laravel/telescope#1051]
- Telescope Version: 4.4.9
- Laravel Version: 8.37.0
- PHP Version: 8.0.3
- Database Driver & Version: MySQL 5.5

### Description:

New fresh install Telescope on Laravel 8 is not showing the page correctly,
I'm only seeing the a big icon of Laravel telescope on the screen also from the console log there is an error message

GET http://localhost/vendor/telescope/app.css?id=d2d43f24f2a54d31e084 net::ERR_ABORTED 404 (Not Found)
GET http://localhost/vendor/telescope/app.js?id=ff7aa0235e43ae0474e1 net::ERR_ABORTED 404 (Not Found)

### Steps To Reproduce:

php artisan cache:clear
php artisan config:clear
php artisan route:clear

On and Before the installation
Tried reinstalling Telescope
Followed the instructions from laravel documentation correctly. 
![image_2021-04-14_234138](https://user-images.githubusercontent.com/26690711/114738866-f52efb80-9d7a-11eb-84b7-42cda567c233.png)

**Comment by driesvints** [comment]
![Uploading Screenshot 2021-04-15 at 10.34.51.jpg…]()

**Comment by monsterdream** [comment]
Indeed, it's stopped working for me also. Same errors. Only thing I changed it was switch to docker runtime since I'm using Laravel Vapor. It's working fine on local development, but not for production anymore.
